monitoring/ceph-mixin/dashboards_out/ceph-application-overview.json

```
@@ -173,14 +173,14 @@
             "uid": "${datasource}"
           },
           "editorMode": "code",
-          "expr": "Sum(rate(ceph_pool_rd{cluster=~\"$cluster\",}[$__rate_interval]) *\n  on(pool_id) group_left(instance,name) ceph_pool_metadata{cluster=~\"$cluster\",name=~\"$pool\"})",
+          "expr": "sum(rate(ceph_pool_rd{cluster=~\"$cluster\",}[$__rate_interval]) *\n  on(pool_id) group_left(instance,name) ceph_pool_metadata{cluster=~\"$cluster\",name=~\"$pool\"})",
           "hide": false,
           "legendFormat": "Read",
           "range": true,
           "refId": "B"
         }
       ],
-      "title": "IOPS",
+      "title": "IOPS - Pool Level",
```

Lets not call it pool level- as we have other metrics we should rename them as well.

Possible solution is to use the $variable to say which applications pool we are showing for. Do it same for throughput panel as well. 
 if its all or particular pool.

IOPS ($application - $pool)

```suggestion
      "title": "IOPS",
```